backend/app/api/ws/chat_namespace.py

```
@@ -1745,6 +1551,206 @@ def get_device_id(task):
+        )
+    else:
+        task_model_id, force_override = extract_model_override_info(task)
+        if force_override and task_model_id:
+            model_id = task_model_id
+            logger.info(
+                f"[WS] chat:retry use_model_override=False, using model from task metadata: "
+                f"model_id={model_id}"
+            )
+        else:
+            logger.info(
+                f"[WS] chat:retry use_model_override=False, no task metadata model, "
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)